app/components/AppToolbar/index.js

```diff
@@ -30,6 +30,7 @@ const styleSheet = createStyleSheet('zmoraAppToolbar', (theme) => ({
   toolbar: {
     transform: 'translate3d(0,0,0)',
     overflow: 'hidden',
+    backgroundColor: '#673BB7',
```

Comment:
@marcskow tak za pomocą globali. Jest coś takiego jak them w material-ui możesz poszukać, jak nie znajdziesz to ja popatrzę tylko chwilę mi zejdzie żeby się ogarnąć dzisiaj.